src/main/java/ecdar/controllers/TransitionPaneElementController.java

```diff
@@ -45,16 +43,16 @@ public class TransitionPaneElementController implements Initializable {
 
     @Override
     public void initialize(URL location, ResourceBundle resources) {
-        Ecdar.getSimulationHandler().availableTransitions.addListener((ListChangeListener<Transition>) c -> {
-            while (c.next()) {
-                for (Transition trans : c.getAddedSubList()) insertTransition(trans);
+        // Ecdar.getSimulationHandler().availableTransitions.addListener((ListChangeListener<Transition>) c -> {
+        //     while (c.next()) {
+        //         for (Transition trans : c.getAddedSubList()) insertTransition(trans);

```

Comment:
why is this commented out?

Reply:
slettet

Reply:
its still there, is it committed?

Reply:
github is being weird on my end, all good my bad